definitions/private-law/json/AuthorisationCaseEvent/ApplicantsDetails/amend/AuthorisationCaseEvent.json

```diff
@@ -12,6 +12,12 @@
           "caseworker-privatelaw-superuser"
         ],
         "CRUD": "CRU"
+      },
```

Comment:
you know that its not only this event which generates the c8, its also "applicantsDetails" event?